resources/datadog-integrations/software.rb.erb

```diff
@@ -13,9 +13,46 @@ build do
 
   # copy files
   mkdir "#{integration_agent_folder}"
-  mkdir "/etc/dd-agent/conf.d/examples/"
-  copy "#{integration_source_folder}/conf.yaml.example", "/etc/dd-agent/conf.d/examples/<%= name %>.yaml.example"
+
+  # The conf directory is different on every system
+  if linux?
+    conf_directory = "/etc/dd-agent/conf.d"
+  elsif osx?
+    conf_directory = "#{install_dir}/etc"
+  elsif windows?
+    conf_directory = "../../extra_package_files/EXAMPLECONFSLOCATION"
+  end
+
+  mkdir "#{conf_directory}/examples/"
+
+  # Move config files to examples folder.
+  if File.exists? "#{integrations_dir}/#{check}/conf.yaml.example"
+    copy "#{integration_source_folder}/conf.yaml.example", "/etc/dd-agent/conf.d/examples/<%= name %>.yaml.example"
+  end
+
+  # If it is a default, we need to move it as a default
+  if File.exists? "#{integrations_dir}/#{check}/conf.yaml.default"
+    copy "#{integration_source_folder}/conf.yaml.default", "/etc/dd-agent/conf.d/examples/<%= name %>.yaml.default"
```

Comment:
This is a real issue. If we change a default and the customer wants to get the last changes, he'll have to change the file manually. I can't think of a good solution right now though...